extensions/amp-ad/0.1/amp-ad-api-handler.js

```diff
@@ -84,6 +88,9 @@ export class AmpAdApiHandler {
     this.iframe_ = iframe;
     this.is3p_ = is3p;
     this.iframe_.setAttribute('scrolling', 'no');
+    let iframeSrc = this.iframe_.getAttribute('src');
+    iframeSrc = iframeSrc.substring(0, iframeSrc.indexOf('#'));
+    this.useApi_ = endsWith(iframeSrc, '.html');
```

Comment:
I was about to check frame.html or `RTV version + frame.html` But I don't want to create difference for localDev mode and test mode, so i only check `.html` here. I do get `this.useApi_ = false` here on an `adsense` ad with A4A experiment on